lib/oban.ex

```diff
@@ -354,15 +354,17 @@ defmodule Oban do
           name(),
           multi :: Multi.t(),
           multi_name :: Multi.name(),
-          changesets_or_wrapper :: [job_changeset()] | %{changesets: [job_changeset()]}
+          changesets_or_wrapper ::
+            [job_changeset()] | %{changesets: [job_changeset()]} | (map -> [job_changeset()])
```

Comment:
This typespec is getting a little out of hand. What if we pull `changesets_or_wrapper` into a type and reference it here instead?